stdlib/multiprocessing/util.pyi

```diff
@@ -0,0 +1,122 @@
+import threading
+from _typeshed import Self, StrOrBytesPath
+from collections.abc import Callable, Iterable, Sequence
+from itertools import count
+from logging import Logger
+from typing import Any
+from weakref import WeakValueDictionary
+
+from . import process
+
+__all__ = [
+    "sub_debug",
+    "debug",
+    "info",
+    "sub_warning",
+    "get_logger",
+    "log_to_stderr",
+    "get_temp_dir",
+    "register_after_fork",
+    "is_exiting",
+    "Finalize",
+    "ForkAwareThreadLock",
+    "ForkAwareLocal",
+    "close_all_fds_except",
+    "SUBDEBUG",
+    "SUBWARNING",
+]
+
+NOTSET: int = ...
+SUBDEBUG: int = ...
+DEBUG: int = ...
+INFO: int = ...
+SUBWARNING: int = ...
+
+LOGGER_NAME: str = ...
+DEFAULT_LOGGING_FORMAT: str = ...
+
+_logger: Logger | None = ...
+_log_to_stderr: bool = ...
+
+def sub_debug(msg: object, *args: object) -> None: ...
+def debug(msg: object, *args: object) -> None: ...
+def info(msg: object, *args: object) -> None: ...
+def sub_warning(msg: object, *args: object) -> None: ...
+def get_logger() -> Logger: ...
+def log_to_stderr(level: int | None = ...) -> Logger: ...
+def _platform_supports_abstract_sockets() -> bool: ...
+def is_abstract_socket_namespace(address: str | bytes | None) -> bool: ...
+
+abstract_sockets_supported: bool = ...
+
+def _remove_temp_dir(rmtree: Callable[[StrOrBytesPath], None], tempdir: StrOrBytesPath) -> None: ...
+def get_temp_dir() -> str: ...
+
+_afterfork_registry: WeakValueDictionary[tuple[int, int, Callable[[Any], Any]], object]
+_afterfork_counter: count[int]
+
+def _run_after_forkers() -> None: ...
+def register_after_fork(obj: object, func: Callable[[Any], Any]) -> None: ...
+
+_finalizer_registry: dict[tuple[int | None, int], Finalize] = ...
+_finalizer_counter: count[int] = ...
+
+class Finalize:
+    _args: Any
+    _kwargs: dict[Any, Any]
+    _callback: Callable[[Any], Any]
+    _key: tuple[int | None, int]
+    _pid: int
+
+    def __init__(
+        self,
+        obj: object | None,
+        callback: Callable[[Any], Any],
```

Comment:
Your annotation currently means "A callable that can only accept a single argument of `Any` type, and returns a value of `Any` type. In this situation, a better annotation is "Any callable with any number of parameters of `Any` type, that returns a value of `Any` type". To express that, we use a `...` in place of a parameter list.

```suggestion
        callback: Callable[..., Any],
```

Reply:
Ah, ok. The various uses of ellipsis sometimes trip me up. Thank you for clarifying that.

Reply:
It annoys me the way `...` is overloaded to mean so many different, unrelated things in the type system, but what can you do 🤷‍♂️